### PR TITLE
#44 with feature branch (Adding Nextcloud)

### PR DESCRIPTION
**Pull request by spacecodeit**
I forgot to use a feature branch in my first PR. #44

### CODE REVIEW

**Comment by luto** [comment]
I'm assuming NextCloud and OwnCloud are still incompatible, so this won't fix #3. Are we happy with OwnCloud or should there be two guides: one for NextCloud, one for OwnCloud?

**Comment by luto** [comment]
Also, thank you very much for the guide! :) :tada:

**Comment by spacecodeit** [comment]
I did a quick test, the steps needed for NextCloud are the same as for ownCloud.

As I know it is also possible to migrate from ownCloud to NextCloud. After a quick research I can't find anything about the other way. The documentation for ownCloud to NextCloud can be found here:
https://docs.nextcloud.com/server/13/admin_manual/maintenance/migrating_owncloud.html

**Comment by tocsinDE** [comment]
> Are we happy with OwnCloud or should there be two guides: one for NextCloud, one for OwnCloud?

For me it's like Gogs vs. Gitea: NextCloud seems to be developed more actively than OwnCloud and gained a lot of traction in the past months, so +1 for NextCloud.

**Comment by nichtmax** [comment]
According to our support, users ask _way more_ often for [Nextcloud](https://wiki.uberspace.de/cool:owncloud). Owncloud seems to be more enterprisey.

**Comment by spacecodeit** [comment]
> According to our support, users ask way more often for Nextcloud. Owncloud seems to be more enterprisey.

If you wan't I can change my guide to Nextcloud, as I mentioned above, the steps seams to be (nearly) the same.

**Comment by ezzra** [comment]
Yes, Nextcloud is much more "sexy" and community driven. But maybe there is no reason why there shouldn't be an ownCloud manual as well? 
I think in the end, there will be several differences in personalization, updates, links etc. that should be reasonable to use separate manuals and not put them together.

**Comment by nichtmax** [comment]
I’ll give my 2 cents tomorrow.

**Comment by nichtmax** [comment]
There is no reason to have both guides, we'd like to have NextCloud first though or both at the same time. @spacecodeit would you like to change this one to NextCloud or should we wait for the NextCloud guide or do you want to clone it and we release both?

**Comment by spacecodeit** [comment]
I will change to NextCloud in the next days.

File: source/guide_nextcloud.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Another thing you can configure to get a better performance is enabling Memcaching.
+
+To enable Memcaching (APCu) you only need to add the following line to your /var/www/virtual/$USER/html/config/config.php:
+
+::
+
+ 'memcache.local' => '\OC\Memcache\APCu',
+
+
```
Comment (nichtmax) [inline source/guide_nextcloud.rst]: The admin interface complains about 
```
The "Strict-Transport-Security" HTTP header is not set to at least "15552000" seconds. For enhanced security, it is recommended to enable HSTS as described in the security tips.
```
Please add a hint that we can't change that at the moment and link to [the manual](https://manual.uberspace.de/en/web-security.html).

File: source/guide_nextcloud.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+============
+
+``cd`` to your `document root`_, then download the latest release of the Nextcloud and extract it:
+
+.. note:: The link to the lastest version can be found at Nextcloud's `download page <https://nextcloud.com/install/#instructions-server>`_.
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ curl https://download.nextcloud.com/server/releases/nextcloud-42.23.1.tar.bz2 | tar -xjf - --strip-components=1
```
Comment (nichtmax) [inline source/guide_nextcloud.rst]: Can you change that to 

```
+ [isabell@stardust html]$ curl https://download.nextcloud.com/server/releases/latest.tar.bz2 | tar -xjf - --strip-components=1
```

please?

**Comment by nichtmax** [comment]
Can you please add the [versions](https://github.com/Uberspace/lab/blob/master/source/README.md#boilerplate) you tested with?

**Pull request merged**